src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,57 @@
-export const Tabs = () => {};
+import React from 'react';
+import classNames from 'classnames';
+import { Tab } from '../../types/Tab';
+
+type Props = {
+  tabs: Tab[];
+  selectedTabId: string;
+  onTabSelected: (tab: Tab) => void;
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const currentTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
```

Comment:
Create helper function in `helpers.ts`

Reply:
I think using helpers for so little project and for one func is useless